src/FilterService.Broker/Requests/RoleService.cs

```diff
@@ -31,21 +31,21 @@ public class RoleService : IRoleService
       _rcGetRoles = rcGetRoles;
     }
 
-    public async Task<List<RoleFilteredData>> GetRolesFilterDataAsync(List<Guid> roleIds, List<string> errors)
+    public async Task<List<RoleFilteredData>> GetRolesFilterDataAsync(List<Guid> rolesIds, List<string> errors)
```

Comment:
maybe rename to GetRolesFilteredDataAsync?

Reply:
done